Assets/Scripts/SS3D/Systems/Interactions/InteractionController.cs

```diff
@@ -28,6 +28,7 @@ public sealed class InteractionController : NetworkActor
         private Camera _camera;
         private RadialInteractionView _radialView;
 
+        [Client]
         protected override void OnStart()
```

Comment:
This method is called on both client and server, and will not execute (and throw a warning) in Server Only mode. Instead of overriding OnStart(), override OnStartClient() to achieve your intent.